BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
+
+### Table of Contents
+
+1. [Grunt Tasks](#grunt-tasks)
+
+## Grunt Tasks
+### clean
+The clean task cleans out the Distribution directory in the FullScreenShenanigans project. This occurs because if a new build has changes made, the Distribution folder must be recreated to reflect those changes.
+
+### tslint
+The tslint task make sure that the .ts files in the project are neat and readable. It checks each file for whitespaces and alignment issues and throws errors if it finds any.
+
+### typescript
+#### typescript:base
+The typescript:base task translates the project's main .ts file into a .js file. 
+#### typescript:distribution
+The typescript:distribution task translates the project's main .ts file into a .js file that is placed in the Distribution directory and concatenated with the version number of the project.
+
+### copy
+The copy:dist task copies files and folders from the Source directory into the Distribution directory.
```

Comment:
Ironically, the task with the most configuration lines has the smallest description.